R/control.ergm.R

```diff
@@ -460,26 +477,30 @@
 #' @keywords models
 #' @export control.ergm
 control.ergm<-function(drop=TRUE,
-
+                       
                        init=NULL,
                        init.method=NULL,
                        
                        main.method=c("MCMLE", "Stochastic-Approximation"),
                        force.main=FALSE,
                        main.hessian=TRUE,
-
+                       
                        checkpoint=NULL,
                        resume=NULL,
-
+                       
                        MPLE.samplesize=.Machine$integer.max,
                        init.MPLE.samplesize=function(d,e) max(sqrt(d),e,40)*8,
                        MPLE.type=c("glm", "penalized","logitreg"),
                        MPLE.maxit=10000,
                        MPLE.nonvar=c("warning","message","error"),
                        MPLE.nonident=c("warning","message","error"),
                        MPLE.nonident.tol=1e-10,
+                       MPLE.covariance.samplesize =500,
```

Comment:
Why this value?

Reply:
Again, no particular reason to not change this value. For the simulations in our paper we set the samplesize = 500 and found that it worked well for us.